### PR TITLE
systemtest: fix `always-incremental-consolidate` sporadic failures

### PR DESCRIPTION
**Pull request by alaaeddineelamri**
#### Description

This PR fixes `ai-consolidate-ignore-duplicate-job` failing sporadically on different systems by adding a small delay
between test job runs. The test is also renamed to `always-incremental-consolidate`

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General
- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [x] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
- [x] Commit descriptions are understandable and well formatted

##### Source code quality

- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

### CODE REVIEW

File: systemtests/tests/CMakeLists.txt [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #   02110-1301, USA.
 
 add_subdirectory(acl)
-add_subdirectory(ai-consolidate-ignore-duplicate-job)
+add_subdirectory(always-incremental-consolidate)
```
Comment (bruno-at-bareos) [inline systemtests/tests/CMakeLists.txt:21]: Maybe you want your last commit be amended to adjust the description systemtest: renamed ai-consolidate-ignore-duplicate-job to always-incremental-consolidate

File: systemtests/tests/always-incremental-consolidate/testrunner [hunk 2]
```diff
@@ -116,8 +118,6 @@ END_OF_DATA
 
 run_bconsole
 
-stop_bareos
```
[the remark this answers is not in the document]
Comment (alaaeddineelamri) [reply systemtests/tests/always-incremental-consolidate/testrunner:119]: When a test is a single testrunner, the `end_test` functions tries to stop bareos. If you keep both `end_test` and `stop_bareos` you'd be trying to stop bareos twice.

File: systemtests/tests/CMakeLists.txt (hunk 1 above)
Comment (alaaeddineelamri) [reply systemtests/tests/CMakeLists.txt:21]: sure!

File: systemtests/tests/always-incremental-consolidate/etc/bareos/bareos-dir.d/job/backup-bareos-fd.conf.in [hunk 3]
```diff
@@ -7,4 +7,8 @@ Job {
   AlwaysIncrementalJobRetention = 1 seconds
   AlwaysIncrementalKeepNumber = 1
   AlwaysIncrementalMaxFullAge = 1 seconds
+  Run Script {
```
Comment (bruno-at-bareos) [inline systemtests/tests/always-incremental-consolidate/etc/bareos/bareos-dir.d/job/backup-bareos-fd.conf.in:10]: Question: The job time start is took into account only after before script is completely done, is this instruction really help us to be on the safe side when time calculation is done ?

Comment (alaaeddineelamri) [reply systemtests/tests/always-incremental-consolidate/etc/bareos/bareos-dir.d/job/backup-bareos-fd.conf.in:10]: What we need to take in consideration is the end time not start time. Shifting the start will definitely shift the end time.

**Review by bruno-at-bareos — APPROVED** [review]
Everything build fine locally and all checks have been done.

**Review by bruno-at-bareos — APPROVED** [review]
Approving last changes.

**Pull request merged**